examples/hpc/dla-driver/src/lib.rs

```diff
@@ -483,46 +530,62 @@ impl Dla {
         }
     }
 
+    /// Reads index of the first input bank
     fn get_input_bank(&self) -> MemoryBank {
         let mut reg = self.read_u32(DLA_BUF_DATA_BANK);
         reg = get_bits!(reg, DLA_BUF_DATA_BANK_B_BITMASK);
-        MemoryBank::from_u32(reg)
+        let bank = match MemoryBank::try_from(reg) {
+            Ok(bank) => bank,
+            Err(e) => panic!("{}", e),
+        };
+        bank
```

Comment:
If there's a way to assume that the register always contains a valid value, you should  use `.unwrap()` here. Say, we assume the hardware doesn't ever change the value and that we make sure to only store valid values with assertion / match on the store side.

Reply:
changed to use `.unwrap()`